docs/config.toml

```diff
@@ -230,12 +236,25 @@ defaultContentLanguage = "en"
     weight = 303
     parent = "authoring-bundles"
 
+[[menu.main]]
+    name = "Distributing Bundles"
+    url = "/distributing-bundles"
```

Comment:
You'll need an entry below this group placeholder placeholder for the distributing bundles. Take a look at the other group headings and you'll see that I've been using "Overview".  When you click on the group, it just expands, then you can click on the "overview" to see the content you wrote for the distributing bundles section. Right now if you check out the preview you'll see that you can't actually navigate to the distributing bundles url.